src/remoted/secure.c

```diff
@@ -242,6 +243,10 @@ void * rem_handler_main(__attribute__((unused)) void * args) {
     int wdb_sock = -1;
     mdebug1("Message handler thread started.");
 
+    // Reset all the agents' connection status in Wazuh DB
+    if (OS_SUCCESS != wdb_reset_agents_connection(&wdb_sock))
+        mwarn("Unable to reset the agents' connection status. Possible incorrect statuses until the agents get connected to the manager.");
```

Comment:
Typo agents'

Reply:
Do you mean the apostrophe location? I think that is correct because when the word ends with `s` (either because of plurality or just because of the word), the apostrophe to indicate the possessive must be located after the `s`.

Reply:
You are right. I missuderstood it